rcl/src/rcl/init.c

```diff
@@ -155,7 +193,6 @@ rcl_init(
   }
 
   TRACEPOINT(rcl_init, (const void *)context);
-
```

Comment:
@ivanpauno spurious change?

Reply:
Solved in https://github.com/ros2/rcl/pull/515/commits/a905ae9dc01d24d4b09f0999f538b61a0e83315e.